configure/CMakeLists.txt

```diff
@@ -1,23 +1,37 @@
 include(FindPkgConfig)
 
 if(NOT CMAKE_BUILD_TYPE)
-    message("No build type specified - default is DEBUG")
-    set(CMAKE_BUILD_TYPE DEBUG)
+    message("No build type specified - default is RELEASE")
+    set(CMAKE_BUILD_TYPE RELEASE)
 endif()
 
 if(IDL_BASE)
-    set(IDL_PKG_INCLUDE_DIRS ${IDL_BASE}/include)
+    message("Using IDL_BASEIDL_BASE=$ENV{IDL_BASE}")
```

Comment:
This is breaking something on Linux.
The user should be able to pass -DIDL_BASE=<IDL_BASE_folder> to cmake without having to specify the IDL_BASE environment variable.
I would suggest to fix the typo in the message and also discourage the use of $ENV...
Can't we use ${IDL_BASE} as it used to be instead of $ENV{IDL_BASE}?

The same applies for OMNI_BASE and ZMQ_BASE below.